content/how-to/general/support/index.md

```diff
@@ -1,12 +1,12 @@
 ---
 permalink: support
-audit_date: '2018-12-13'
+audit_date: '2021-06-07'
```

Comment:
It hasn't been audited, though, right?

Reply:
I went through all the numbers with the SME